src/__tests__/App.test.js

```diff
@@ -18,21 +15,21 @@ describe('App', () => {
 
     it('addTotal', () => {
       const fakeEvent = {target: {checked: 'true', id: '2'}}
-            // console.log(app.state);
       app.addTotal(fakeEvent)
-            // const expected = {product1: true};
-      expect(app.state.total).toEqual(10)
-            // expect(app.state.addTotal).toEqual({expected});
+      expect(app.state.total).toEqual(9.99)
     })
 
     it('properly decrements price', () => {
       wrapper.setState({total: 10})
       const fakeEvent = {target: {value: ':not(:checked)', id: '2'}}
-      // const fakeId = {target: {id: '2'}}
       app.addTotal(fakeEvent)
-      // console.log(app.state.total)
       expect(app.state.total).toEqual(0)
-            // const expected = {product1: true};
+    })
+  })
+  describe('roundUp', () => {
+    it('will roundUp correctly', () => {
+      let output = roundUp(359.96000000000004)
+      expect(output).toEqual(359.96)
```

Comment:
The test says `'will roundUp correctly'`, but the test is actually showing an example of rounding *down*. You might want to change the `it` text to better match the situation, and you may want to also write a test-case for the round-up use case